src/components/rockets/Rockets.js

```diff
@@ -1,7 +1,19 @@
-const Rockets = () => (
-  <div className="rockets">
-    <h1>Rockets Under Construction</h1>
-  </div>
-);
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { getRockets } from '../../redux/rockets/rockets';
+import Rocket from './Rocket';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getRockets());
+  }, []);
```

Comment:
Just as an informative comment: this gets run every time the component gets mounted. That is, if you change to the missions page and then change back to the rockets page, the action will get dispatched again.
If you expect this to only run once and get weird bugs, this might be the cause.

Reply:
thank you for your observation, I used this useEffect just to test that my fetching function works good ^^